cluster/examples/kubernetes/ceph/cluster-test.yaml

```diff
@@ -36,6 +36,9 @@ spec:
   mgr:
     count: 1
     allowMultiplePerNode: true
+  network:
+    connectionsEncrypted: true
+    connectionsCompressed: true
```

Comment:
Should these be set in `cluster-test.yaml` for everyone?

Reply:
We might leave it for testing, but I think i'll revert it.